obs/CMakeLists.txt

```diff
@@ -18,20 +18,27 @@ endif()
 option(ENABLE_BACKTRACE "Enable backtrace support on crash" ON)
 add_feature_info(ENABLE_BACKTRACE ENABLE_BACKTRACE "Backtrace support.")
 
-add_compile_options(
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

Comment:
Please split this up into separate commits, this has nothing to do with the installation path.

Reply:
Unless I messed up these should be separate commits, though I can pull it out into another PR if you'd prefer.

Reply:
No, this is part of the one commit, please split it up.

Reply:
I just removed the second commit from this PR, I'll continue to experiment with my local install in [another branch](https://github.com/arcnmx/LookingGlass/compare/obs-cmake...arcnmx:obs-cmake2). None of it is all that relevant anyway, I was just trying to clean up a bit and tie the flags to targets to make it easier to customize.